mockserver-core/src/test/java/org/mockserver/server/initialize/ExpectationInitializerLoaderTest.java

```diff
@@ -314,11 +314,11 @@ public void shouldLoadExpectationsFromFileSystemInJsonOnWithGlobWithStar()  thro
                 )
             };
             File mockserverInitializerThree = File.createTempFile(uniquePrefix + "_mockserverInitializationThree", ".json");
-            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectionsThree).getBytes(StandardCharsets.UTF_8));
+            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectationsThree).getBytes(StandardCharsets.UTF_8));
             ConfigurationProperties.initializationJsonPath(mockserverInitializer.getParentFile().getAbsolutePath() + "/" + uniquePrefix + "_mockserverInitialization*.json");
 
             // when
-            final Expectation[] expectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
+            expectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
```

Comment:
As the code that follows doesn't seem to care about the earlier one, I'm just removing `final`.

I could rename it instead. 

It also looks like the variables could be given distinct block scopes to avoid the rename, but the cost of reindenting the two sections would be high...